dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -426,24 +470,43 @@ public synchronized void initialize(Service service) {
                 }
             }
             for (DOMSource r : schemas) {
+                if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
```

Comment:
Maybe continue using isLoggableFine instead of checking again?

Reply:
yes, will check boolean